zipline/__main__.py

```diff
@@ -227,9 +235,18 @@ def run(ctx,
         broker,
         broker_uri,
         state_file,
+        broker_list,
```

Comment:
Please move this variable after the `realtime_bar_target` to be in sync with the parameter order above.

Reply:
Also, please rename this variable to list_brokers.